standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStore.java

```diff
@@ -8322,6 +8322,22 @@ public AllocateTableWriteIdsResponse allocate_table_write_ids(
       return response;
     }
 
+    @Override
+    public MaxAllocatedTableWriteIdResponse get_max_allocated_table_write_id(MaxAllocatedTableWriteIdRequest rqst)
```

Comment:
why not Camel case?

Reply:
All the functions in HMS looks like this I don't want to break the pattern. On the second glance, I had to change the seedWriteId and seedTxnId to look like this...